platform/aot_android_arm.go

```diff
@@ -0,0 +1,32 @@
+package platform
+
+/*
+#include "vm_def.h"
+*/
+import "C"
+
+import (
+	"github.com/perlin-network/life/exec"
+	"unsafe"
+)
+
+type AOTContext struct {
+	dlHandle unsafe.Pointer
+	vmHandle *C.struct_VirtualMachine
```

Comment:
The `dlHandle` and `vmHandle` fields can be removed (so that cgo is no more needed).